modules/microshift-nw-multus-add-pod.adoc

```diff
@@ -73,7 +73,7 @@ metadata:
 ----
 $ oc apply -f ./<test-bridge>.yaml <1>
 ----
-<1> Replace _<test-bridge>_ with the pod name that you want to use.
+<1> Replace `<test-bridge>` with the pod name that you want to use.
```

Comment:
```suggestion
<1> Replace `<test_bridge>` with the pod name that you want to use.
```

Reply:
here the value <test-bridge> is found in 10 places. i will have to replace it as <test_bridge? in all 10 places to maintain consistency. It cannot be changed only in line 76

Reply:
@DanielFroehlich please confirm if the hyphen can be replaced with _ in the lines 76 and all other places in this file?

Reply:
@aravipra This is @pmtk's content. The underscore is docs formatting, not an engineering issue. 

The rule is to use italics and underscores with the angle brackets in all user-replaceable values. In this case, the original OCP module used both user-replaceable AND examples values. I didn't update those; but since you're doing this optimization work, I think it's a good idea to use all user-replaceable values to make the module consistent. Go ahead with adding the italics in the code blocks, so _<test_bridge>_ and the same plus backticks in text, so `_<test_bridge>_`.

Reply:
OCP has an exception to this SSG rule in their docs because the rule was added years into their docs and they decided NOT to follow it. (Hence the discrepancy in peer reviews we get.) MicroShift aims to follow the rule.

Reply:
Thanks, @ShaunaDiaz . Do we need to remove from the OCP guidelines doc:

![Screenshot from 2024-09-03 17-30-34](https://github.com/user-attachments/assets/241f31fc-159b-4eb4-9ba5-4830cf69193f)

Reply:
No, because the OCP do not want to backport the style update, hence the exception. MicroShift lives in the repo, but is published separately, so it _does_ adhere to the guidelines. Because these are two docsets, should be OK to let each do their own on this one. (Just like MicroShift uses "offline" as an exception, but OCP does not.)

Reply:
done